lua/luminate/highlight.lua

```diff
@@ -10,6 +10,11 @@ end
 
 function M.on_bytes(event_type, bufnr, changedtick, start_row, start_column, byte_offset, old_end_row, old_end_col,
                     old_end_byte, new_end_row, new_end_col, new_end_byte)
+  if #vim.api.nvim_get_option_value('buftype', { buf = 0 }) ~= 0 then
```

Comment:
This code effectively addresses issues with plugins like Lazy and Mason. However, in my setup, it didn't have the desired effect on sidebar.nvim (https://github.com/sidebar-nvim/sidebar.nvim).

I'll give this some more thought to see how it can be resolved